src/helpers/balloonHelper.ts

```diff
@@ -1,36 +1,40 @@
+//@ts-nocheck
```

Comment:
why?

Reply:
Because i have an error on the function getPixelFromCoordinate : "Property 'getPixelFromCoordinate' does not exist on type 'OpenlayersMap'"

I take this function from the file balloonHelper.js on map-ui, and in their function they use getPixelFromCoordinate like me. I will ask to VC System why this error is display.

In the meantime, I will add a comment to detail the presence of the @ts-nocheck